src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -193,7 +193,7 @@ struct PropertyName : public std::string {
      * @return true if property is mutable
      */
     bool is_mutable() const {
```

Comment:
could you please also update this place https://github.com/openvinotoolkit/openvino/blob/70ec5317f43a2576ec4aef08c91671b6beacfb47/src/bindings/python/src/pyopenvino/utils/utils.cpp#L188-L196 ?

Maybe not directly related to this PR, but since you changing common code .. we can finally fix `is_mutable` and its usage.

Reply:
Done here: https://github.com/openvinotoolkit/openvino/pull/28907